stack-nightly.yaml

```diff
@@ -0,0 +1,12 @@
+resolver: nightly-2019-10-31
+
+packages:
+  - summoner-cli/
+  - summoner-tui/
+
+extra-deps:
+  - base-noprelude-4.13.0.0
+  - brick-0.50
+  - generic-deriving-1.12.4
```

Comment:
Will we use newer `generic-deriving` under separate PR? We have an issue for that. I see that `brick` and `relude` were updated. Can do this with `generic-deriving` as well... https://github.com/kowainik/summoner/issues/337
What do you think?